### PR TITLE
model: support columar table

### PR DESCRIPTION
**Pull request by youjiali1995**
Signed-off-by: youjiali1995 <zlwgx1023@gmail.com>

### What problem does this PR solve? 
Support columnar table.

### What is changed and how it works?
Add a `IsColumnar` field to `TableInfo`. It's true when the engine is `TiFlash`.

### Check List 

Tests 

 - No code

Code changes

 - Has exported variable/fields change

### CODE REVIEW

File: model/model.go [hunk 1]
```diff
@@ -312,6 +312,10 @@ type TableInfo struct {
 
 	// TiFlashReplica means the TiFlash replica info.
 	TiFlashReplica *TiFlashReplicaInfo `json:"tiflash_replica"`
+
+	// IsColumnar means the table is column-oriented.
+	// It's true when the engine of the table is TiFlash.
```
Comment (wjhuang2016) [inline model/model.go:317]: Is TiFlash Only?

Comment (youjiali1995) [reply model/model.go:317]: Yes

Comment (wjhuang2016) [reply model/model.go:317]: If so, I think it's better to say "It's true when the engine of the table is TiFlash only."

**Comment by cfzjywxk** [comment]
LGTM

**Comment by youjiali1995** [comment]
🤨Why the test always fails? I just added a field.

**Comment by kennytm** [comment]
no need to care about build-integration, it should be fixed from the tidb side.

**Comment by cfzjywxk** [comment]
Seems some planner tests have failed, still unstable by now...

**Review by wjhuang2016 — COMMENTED** [review]
LGTM

**Comment by youjiali1995** [comment]
@wjhuang2016 Can you merge it?

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**